ci/make_benchmark_ssr_cmt.py

```diff
@@ -16,11 +16,12 @@ def write_benchmark(lines: List[str], content: List[Dict[str, str]]) -> None:
 
     for i in content:
         lines.append(
-            "| {i.name} | {i.round} | {i.min} | {i.max} | {i.mean} | {i.std_dev} |"
+            f"| {i.name} | {i.round} | {i.min} | {i.max} | {i.mean} | {i.std_dev} |"
```

Comment:
To use the dicts as objects (with `.name` instead of `["name"]`) I think you can apply [this technique](https://stackoverflow.com/a/15882054)